src/ui/public/persisted_state/persisted_state.js

```diff
@@ -272,7 +250,35 @@ export default function (Private) {
     if (!silent && stateChanged) this.emit('change');
 
     return this;
-  };
+  }
+}
+
+function validateParent(parent, path) {
```

Comment:
Was there a reason for leaving these helper functions at the bottom but puting `parentDelegationMixin` above the class? I don't really care which you prefer, but it's nice when all the helpers are kept together.